data/avro-schema/src/main/resources/avro/net/corda/data/membership/actions.request/DistributeMemberInfo.avsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "DistributeMemberInfo",
+  "namespace": "net.corda.data.membership.actions.request",
+  "doc": "Distribute an updated member info to the rest of the network.",
+  "fields": [
+    {
+      "name": "mgm",
+      "doc": "The membership group manager of the group.",
+      "type": "net.corda.data.identity.HoldingIdentity"
+    },
+    {
+      "name": "updatedMember",
+      "doc": "The member which has been updated.",
+      "type": "net.corda.data.identity.HoldingIdentity"
+    },
+    {
+      "name": "minimumGroupParametersEpoch",
+      "doc": "The minimum group parameter epoch to be distribute (if null we use the latest version). If the group parameters with this epoch hasn't been published, then the membership actions processor will requeue this request to retry later.",
+      "type": ["null", "int"]
+    },
+    {
+      "name": "minimumUpdatedMemberSerial",
+      "doc": "The serial number of the minimum version of the updated members' memberInfo to distribute (if null we use the latest version). If the member info with this serial hasn't been published, then the membership actions processor will requeue this request to retry later.",
+      "type": ["null", "long"]
```

Comment:
I don't really see how or in which case this can be null. As far a Is understand we get this epoch from the member info, where serial is non-nullable. Or is this nullability part of some future plans which I missed?
Actually, same goes for epoch. Are we sure there is a case when it can be null? Because I think we are throwing an exception in case it ends up being null.

Reply:
The idea is that you set it too null if you don't care which version you distribute (but there has to be a version to distribute). This probably is only useful if you updated the member info and not the group parameters.

Reply:
As I can see in ApproveRegistrationHandler we will still use the current epoch, so it technically won't be null. This is not a problem I think, but I feel like we are doing null checks on a field which can only be null in theory, but actually never be.

Reply:
Actually what I mean is: This probably is only useful if you updated the group parameters and not the member info.

Reply:
I think we can set the `minimumGroupParametersEpoch` to null in the ApproveRegistrationHandler, if the member if not a notary.

Reply:
Discussed offline and I'm happy to go with nullables.